src/locales/hu.yaml

```diff
@@ -101,6 +101,7 @@ app:
       add_file: Fájl hozzáadása
       command_palette: Parancs paletta
       download_file: Fájl lekérése
+      keyboard_shortcuts: Billentyűparancs
```

Comment:
I think this one has been removed from the en.yaml file a while ago, so it is safe to remove it from here too.